lib/tasks/organization_codes.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "csv"
+
+namespace :organization_codes do
```

Comment:
This implies that the codes can not be added nor from System, neither from the Admin panel?